test/net/sourceforge/kolmafia/objectpool/ConcoctionTest.java

```diff
@@ -22,70 +24,76 @@ public class ConcoctionTest {
   // properly handled by the compareTo.
   @Test
   public void itShouldSortUsables() {
-    LockableListModel<Concoction> usableList = ConcoctionDatabase.getUsables();
-    int thing = usableList.size();
-    usableList.sort();
-    assertEquals(usableList.size(), thing);
+    var useables = ConcoctionDatabase.getUsables();
+    int sizeBeforeSort = useables.size();
+    useables.sort();
+    assertThat(useables.size(), equalTo(sizeBeforeSort));
+  }
+
+  @Test
+  public void steelOrgansAreSortedToTheTop() {
+    var steelMargarita = ConcoctionPool.get(ItemPool.STEEL_LIVER);
+    var nonSteelOrgan = ConcoctionPool.get(ItemPool.PERFECT_NEGRONI);
+    assertThat(nonSteelOrgan, notNullValue());
+    assertThat(steelMargarita.compareTo(nonSteelOrgan), is(-1));
+    assertThat(nonSteelOrgan.compareTo(steelMargarita), is(1));
+  }
+
+  @Test
+  public void steelOrgansThemselvesAreSortedByName() {
+    var steelMargarita = ConcoctionPool.get(ItemPool.STEEL_LIVER);
+    var steelLasagna = ConcoctionPool.get(ItemPool.STEEL_STOMACH);
+    assertThat(steelMargarita.compareTo(steelLasagna), is(1));
+    assertThat(steelLasagna.compareTo(steelMargarita), is(-1));
   }
 
   @Test
   public void exerciseSameNames() {
     // Exercise compareTo and equals for Concoctions with the same name
-    // Lazy way to get two Concoctions with the same name
-    LockableListModel<Concoction> usables = ConcoctionDatabase.getUsables();
-    List<Concoction> Eds = new ArrayList<>();
-    Iterator<Concoction> ui = usables.iterator();
-    while (ui.hasNext()) {
-      Concoction x = ui.next();
-      if (x.toString().contains("Eye of Ed")) {
-        Eds.add(x);
-      }
-    }
-    assertEquals(2, Eds.size());
-    Concoction e1 = Eds.get(0);
-    Concoction e2 = Eds.get(1);
+    var e1 = ConcoctionPool.get(2286);
+    var e2 = ConcoctionPool.get(7962);
+
```

Comment:
A comment here explaining the magic numbers 2286 and 7962 would be helpful.

Reply:
Better yet, use the constants from ItemPool.:

```
  public static final int EYE_OF_ED = 2286;
  public static final int ED_EYE = 7962;
```